client/contivModelClient.go

```diff
@@ -64,14 +64,70 @@ package client
 
 import (
 	"bytes"
+	"crypto/tls"
 	"encoding/json"
 	"errors"
 	"io/ioutil"
 	"net/http"
+	"os"
+	"os/user"
 
 	log "github.com/Sirupsen/logrus"
 )
 
+var httpClient *http.Client
+
+func init() {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+
+	httpClient = &http.Client{Transport: tr}
+}
+
+type NetctlConfig struct {
+	Token string `json:"token"`
+}
```

Comment:
Let's separate out the netctl specific items and move them out of the model as discussed in the modelgen PR.